client/components/layout/stylesheets/sidebar.styl

```diff
@@ -64,16 +65,18 @@
     width 50px
     display inline-block
     margin-bottom 10px
-    opacity 0.7
-    transition opacity 0.3s
+    filter grayscale(100%)
+    -webkit-filter grayscale(100%)
+    transition filter 0.3s
+    -webkit-transition -webkit-filter 0.3s ease
```

Comment:
Hmm I wonder if [nib](https://github.com/tj/nib) supports this transparently, in which case you should be able to get away with `filter grayscale(100%)` and `transition filter 0.3s ease`. Not going to hold up a merge for it though.